libraries/hds/Makefile.am

```diff
@@ -49,6 +49,7 @@ hds_test_LDADD = libhdsf.la
 
 hdsTest_SOURCES = hdsTest.c
 hdsTest_LDADD = libhds.la
+hdsTest_CFLAGS = $(AM_CFLAGS) -DHDS_INTERNAL_INCLUDES
```

Comment:
@grahambell  Why is -DHDS_INTERNAL_INCLUDES needed here, given it is supposed to be testing the installed stuff?